### PR TITLE
Accessibility documentation

### PR DESCRIPTION
**Pull request by jocegonz**

### CODE REVIEW

File: content/xr_design_and_accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+| Font size | <li>Small</li><li>Medium</li><li>Large</li> | Set the default size to medium and allow the user to adjust font size (either by creating preset options or scale) to their comfort level.  This should start at 10% of the screen’s height and adjust as needed. |
+| Friendly fonts | <li>Arial</li> <li>OpenDyslexic</li> | Sans serif and monospaced fonts are often easier to read. Allowing a user to choose a preferred font based on their needs means that their experience is less likely to be diminished by something that we (the devs) can easily adjust. |
+| Visual indicators | <li>Arrows</li> <li>Animations</li> | Due to the nature of the 360 degree experience, we often lack control over what the user is seeing. If there is text that needs to be read, cue the user with visual indicators if they’re looking in the wrong direction. |
+| Alignment | Left-aligned and no justification | Left-aligned text with no justification is easier to read for those with dyslexia. |
+
+# Subtitles
```
Comment (timgcarlson) [inline content/xr_design_and_accessibility.mdx]: Would it be more accurate to title this section "Closed Captioning"?

File: content/xr_design_and_accessibility.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+While most of the recommendations for subtitles will fall under the text category, consider any audio and the cues that might be missed for a person with a hearing disability or low volume settings. Is your experience the same without audio entirely? Would it benefit from captioning?
+
+# Display
+| | What to Consider |
+|:-----|-----:|
```
Comment (timgcarlson) [inline content/xr_design_and_accessibility.mdx]: Same as the other table, the second column may read better left-aligned

File: content/xr_design_and_accessibility.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+| Brightness | Brightness presets can assist people with low vision. |
+| Contrast | Contrast presets can assist people with low vision. Verify if a change in contrast still offers the same amount of depth. |
+| Distance | Consider allowing functionality to bring objects closer. This can help people with low vision or limited mobility. |
+| Size | Size presets can include changing the size of fonts, objects, buttons, etc. Some people would benefit from interacting with a bigger hitbox. |
+
+# Inputs
+Input alternatives can greatly affect those with limited mobility, whether it is situation, temporary or permanent.
+
+|| <div style="width:300px">Alternatives</div> |
+|:-----|-----:|
```
Comment (timgcarlson) [inline content/xr_design_and_accessibility.mdx]: Same as the other table, the second column may read better left-aligned

File: content/xr_design_and_accessibility.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Although we often consider the ideal user, most people do not strictly fall into these categories. Allowing people to choose how they experience their experiences does not only benefit people with disabilities, it benefits everyone. While there may be the inclination to make your experience a literal extended version of reality, we also have the opportunity to create an extended reality that adapts to the needs of the people living in it. People will need different options based on their needs. The ability to adjust preset options allows the user to experience your application in a manner that is comfortable to them. 
+
+This is not a clear cut guide of the do’s and don’ts when designing user experiences in VR, but  a compilation of our research and resources on the subject. With everything, test and see if it works for you and your needs.  
+
+# Text
+|| <div style="width:250px">Options</div> | Why |
+|:-----|:-----:|-----:|
```
Comment (timgcarlson) [inline content/xr_design_and_accessibility.mdx]: The 3rd column (why) appears to be right-aligned. to respect the last row on alignment, I'd say this column should be left-aligned 🤪

File: content/xr_design_and_accessibility.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+| | What to Consider |
+|:-----|-----:|
+| Color | Is color being used to indicate important behavior, like red for “stop” or green for “go”? Consider supplementing these behaviors with patterns, symbols or shapes to enforce differences. |
+| Brightness | Brightness presets can assist people with low vision. |
+| Contrast | Contrast presets can assist people with low vision. Verify if a change in contrast still offers the same amount of depth. |
+| Distance | Consider allowing functionality to bring objects closer. This can help people with low vision or limited mobility. |
+| Size | Size presets can include changing the size of fonts, objects, buttons, etc. Some people would benefit from interacting with a bigger hitbox. |
+
+# Inputs
+Input alternatives can greatly affect those with limited mobility, whether it is situation, temporary or permanent.
```
Comment (timgcarlson) [inline content/xr_design_and_accessibility.mdx]: Should "situation" be "situational"?

File: content/xr_design_and_accessibility.mdx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+|| <div style="width:300px">Alternatives</div> |
+|:-----|-----:|
+| Head movement | Directional input (moving the joystick left or right) <li>Quick turns (rotating 30-45 degrees)</li> <li>Sharp turns (rotating 30-45 degrees with no transition</li> Audio commands |
```
Comment (timgcarlson) [inline content/xr_design_and_accessibility.mdx]: Missing a close parens at `Sharp turns (rotating 30-45 degrees with no transition...` 

image

File: content/xr_design_and_accessibility.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+| Locomotion | <li>Teleportation</li> <li>Free locomotion (free movement within one’s playspace)</li> <li>Rails (moves through the experience for the user. Can cause nausea if emulating real speed)</li> |
```
Comment (timgcarlson) [inline content/xr_design_and_accessibility.mdx]: `Rails (moves through the experience for the user. Can cause nausea if emulating real speed)`

Would read better as:

`Rails (moves through the experience for the user; can cause nausea if emulating real speed)`

**Review by timgcarlson — CHANGES_REQUESTED** [review]
This all looks great! I even learned some new things from it, so thank you!

Just some minor changes and then it's good for me.

**Comment by jocegonz** [comment]
[Compile accessibility work into MAIT](https://trello.com/c/gtU2Yb4p/186-compile-accessibility-work-into-mait)

File: content/xr_design_and_accessibility.mdx (hunk 1 above)
Comment (jocegonz) [reply content/xr_design_and_accessibility.mdx]: Totally. Will update!

File: content/xr_design_and_accessibility.mdx (hunk 2 above)
Comment (jocegonz) [reply content/xr_design_and_accessibility.mdx]: 😩 Building this locally didn't right-align the text, so it didn't register that that was what the colons were doing. I'll just remove them. lol

File: content/xr_design_and_accessibility.mdx (hunk 4 above)
Comment (jocegonz) [reply content/xr_design_and_accessibility.mdx]: super *weird* behavior.

File: content/xr_design_and_accessibility.mdx (hunk 5 above)
Comment (jocegonz) [reply content/xr_design_and_accessibility.mdx]: Updating! Good catch!

File: content/xr_design_and_accessibility.mdx (hunk 6 above)
Comment (jocegonz) [reply content/xr_design_and_accessibility.mdx]: nice.

**Review by timgcarlson — APPROVED** [review]
🙌

**Pull request merged**